lib/generators/hyrax/templates/db/seeds.rb

```diff
@@ -62,7 +62,7 @@ def create_collection(user, type_gid, id, options)
   col = Collection.new(id: id)
   col.attributes = options.except(:visibility)
   col.apply_depositor_metadata(user.user_key)
-  col.collection_type_gid = type_gid
+  col.collection_type_gid = type_gid.to_s
```

Comment:
The `.to_s` appears to be unnecessary because it is called in `collection_type_gid=`.

Reply:
thanks. not sure how this snuck in on me